services/vision/mlvision/ml_model_test.go

```diff
@@ -167,7 +167,7 @@ func TestNewMLDetector(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, gotDetections[0].Score(), test.ShouldBeGreaterThan, 0.789)
 	test.That(t, gotDetections[1].Score(), test.ShouldBeGreaterThan, 0.7)
-	test.That(t, gotDetections[0].BoundingBox().Min.X, test.ShouldBeGreaterThan, 125)
+	test.That(t, gotDetections[0].BoundingBox().Min.X, test.ShouldBeGreaterThan, 124)
```

Comment:
This, and the similar change below, were due to tests failing on my desktop. TFlite tolerances were just too tight for the variance between CPUs. This was the one non-lint change.

Reply:
@bhaney Can you take a quick glance at these two lines? I don't think this should be an issue since it's really testing a range already, but just to be safe.

Reply:
@bhaney I just realized I'd tagged you here (without requesting a full review) and then merged anyway. Sorry about that. Let me know if you see a problem (or if I'm tagging the wrong "vision" team again.)